p/p7zip/stone.yaml

```diff
@@ -0,0 +1,25 @@
+name: 7zip
+version: 24.09
+release: 1
+summary: 7-Zip command-line version for Linux
+license: LGPL-2.1-or-later
+license: BSD-3-Clause
```

Comment:
There should only be one license: 
Delete it and add enough spaces to align to the start of the lgpl one